libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/persistence/Utils.java

```diff
@@ -0,0 +1,42 @@
+package com.linkedin.openhouse.datalayout.persistence;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import com.linkedin.openhouse.datalayout.layoutselection.DataLayoutOptimizationStrategy;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.spark.sql.SparkSession;
+
+/** Utility class for strategies persistence. */
+public final class Utils {
```

Comment:
Utils? :) 
Seems like a Strategy DTO/DAO class.

Reply:
+1 for specific naming.

Reply:
thanks, introduced dao class